MandelaK/SendIT-API
# #[161709465] Created endpoints for getting all deliveries

#### What does this PR do?
Added endpoints allowing users to get all deliveries in our database and ensured they pass the tests

#### Description of Task to be completed?
We are to add API endpoints that allow users get a list of all deliveries in our database. This has been achieved and we get the list whenever the endpoint is accessed

#### How should this be manually tested?
After cloning the repo and installing all dependencies in requirements.txt you can start the app and see that accessing the endpoints GET /api/v1/parcels returns a list of all deliveries in the database.

#### Any background context you want to provide?
No

#### What are the relevant pivotal tracker stories?
#161709465
